src/ripple/app/ledger/LedgerConsensus.cpp

```diff
@@ -32,6 +32,7 @@
 #include <ripple/app/misc/Validations.h>
 #include <ripple/app/tx/TransactionAcquire.h>
 #include <ripple/app/tx/InboundTransactions.h>
+#include <ripple/app/ledger/TxHold.h>
 #include <ripple/basics/CountedObject.h>
 #include <ripple/basics/Log.h>
 #include <ripple/core/Config.h>
```

Comment:
Not your code, but virtuals overridden by `LedgerConsensusImp` should be marked `override`.  I didn't do a full accounting, but examples include `startup()` and `getJson()`.